spec/gateways/security_alert_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe Gateways::SecurityAlert do
+  context 'with no security alerts for application' do
+    it 'returns an empty array' do
+      FileUtils.cp('spec/fixtures/gemfile_with_no_security_alerts', 'tmp/signon_gemfile.lock')
+      result = described_class.new.execute(application_name: 'signon')
+
+      expect(result).to eq([])
+    end
+  end
+
+  context 'with security alerts for application' do
+    context 'with no criticality' do
+      it 'returns an empty list' do
+        FileUtils.cp('spec/fixtures/gemfile_with_nil_criticality', 'tmp/signon_gemfile.lock')
```

Comment:
Wonder if we made the argument for the gateway a path to the gem lock file, we wouldn't have to copy these files? Unless I'm missing something